src/lib/features/project-environments/environment-store.ts

```diff
@@ -112,24 +117,35 @@ export default class EnvironmentStore implements IEnvironmentStore {
     constructor(
         db: Db,
         eventBus: EventEmitter,
-        { getLogger, isOss }: Pick<IUnleashConfig, 'getLogger' | 'isOss'>,
+        {
+            getLogger,
+            isOss,
+            flagResolver,
+        }: Pick<IUnleashConfig, 'getLogger' | 'isOss' | 'flagResolver'>,
     ) {
         this.db = db;
         this.logger = getLogger('db/environment-store.ts');
         this.isOss = isOss;
+        this.flagResolver = flagResolver;
         this.timer = (action) =>
             metricsHelper.wrapTimer(eventBus, DB_TIME, {
                 store: 'environment',
                 action,
             });
     }
 
+    private allColumns() {
```

Comment:
we have previous PR with a migration so this code should never run before the migration is applied. I decided to be extra careful anyway so we have extra protection with a flag before we start asking for required_approvals in the explicit query